docs/7/7.1-configuration-managers.md

```diff
@@ -0,0 +1,81 @@
+# Configuration Managers
+
+### Overview
+
+Configuration Managers (C.M.) are important even in this day of containerization and virtualization. They can be used to set up hosts, create Docker images, create AMIs, and more. Many companies are not yet comfortable with the idea of Docker so this is a nice step in that direction. 
+
+At the end of this section you will: 
+
+- Understand the benefits of configuration management
+- Be able to discuss the main differences and pro/cons of configuration management
+- Compare configuration management with containerization
+- Describe the benefits of immutable infrastructure
+- Be able to explain Infrastructure as Code. 
+
+### Steps
+
+1. Understand Configuration Management
+2. Configuration Management versus Containerization
+3. Immutable Infrastructure
+4. Creating Images with C.M. 
+5. Using Chef with AWS 
+6. Using Ansible with Vagrant
+
+### Understand Configuration Management
+
+Digital Ocean has [An Introduction to Configuration Management](https://www.digitalocean.com/community/tutorials/an-introduction-to-configuration-management) tutorial that is worth reading. Read this before the initial discussion with the leader. 
+
+When reading keep this in mind:
+
+- What is Config Management?
+- How is it useful?
+- Why is it important to new age software engineering?
+- Why use infrastructure as code?
+- Why might some people be against the technology?
+- What are the pros/cons?
+
+### Configuration Management vs Containerization
+
+As you have already learned, containerization is a powerful idea that allows companies to ship pieces of their applications. Historically this type of work was done with C.M. tools but containers do some things better. Containers are built to be repeatable and self contained.
+
+When reading keep this in mind:
+
+- Can containerization and configuration management live together?
+- What does abstracting layers of complexity mean?
+- What is configuration drift?
+
```

Comment:
These "keep in mind" sections don't make as much sense in the format. Lets move them to a deliverables section at the end of the doc to be more in line with the rest of the dob